app/templates/confirmation.html

```diff
@@ -1,22 +1,47 @@
-{% extends theme('layouts/_survey.html') -%}
+{% extends theme('layouts/_survey.html') %}
 
-{% block page_title -%}Summary{% endblock -%}
+{% block page_title %}Confirmation{% endblock %}
 
-{% block main -%}
+{% block main %}
 
-<a class="u-dib" href="{{previous_location}}">Previous</a>
+<div class="grid">
```

Comment:
This needs to extend an existing template an override where necessary rather than duplicate one.